### DISCUSSION TITLE
Gruvbox-Dark theme - Request for Faizul Anwar

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2382] [Show and tell]
This was requested in OFTC's IRC chat room and completed over the weekend. I'm not sure it's pixel perfect and have not been able to get a -Light variant done, but the -Dark one is finished and includes a palette and an Openbox theme as well. Can be downloaded <a href=https://github.com/AzumaHazuki/lxqt-themes-gruvbox>here</a>.

![LXQt-Gruvbox-Dark](https://user-images.githubusercontent.com/67122280/219981562-af25f59a-38bd-4682-9c98-a0c5705fbb7d.png)

**Comment by Butterfly** [comment]
> I'm not sure it's pixel perfect and have not been able to get a -Light variant done

Your timing is perfect! Just a few hours ago, I finally got my Debian to boot via GRUB. It involved a several year old programming bug. One of the _first_ things I found was that no themes **_JUST LIKE THIS_** were instantly available. I need that kind of a more subtle dark gray contrast for my level of vision. I'm a "semi-power user" who gets into all kinds of trouble on my computer every day so I'll try testing that for you (and me!) with respect to any potentially not yet perfect pixels. See you all out here. :grin:

**Comment by edwardp3** [comment]
This looks good. Only difference on my system is that the icons in the window title bar are reversed, the app icon is on the left and the min/max/close icons are on the right.

It also looks similar to [Nord Openbox](https://www.box-look.org/p/1416095/), except in Gruvbox, the title bar is darker, the app icon is larger and the other three icons are smaller.
